moviepy/video/io/ffmpeg_tools.py

```diff
@@ -66,4 +66,30 @@ def ffmpeg_resize(video,output,size):
              output]
              
     subprocess_call(cmd)
+    
+def ffmpeg_stabilize_video(filename, output = None):
+    """
+
+    Makes a new video file, from the given one
+    which is more stable
+
+    Parameters
+    -----------
+
+    filename
+    The name of the shaky video
+
+    output
+    The name of new stabilazed video
+    """
+
+    name, ext = os.path.splitext(filename)
+    if not output:
+        output = "%s_stabilazed%s"%(name, ext)
```

Comment:
The old style should be sufficient, as it'll conform to the styling throughout this file for now.